### ISSUE TITLE
Results for running ‘image_sample.py’ is noise but not the clear image

### ISSUE THREAD

**Issue by fyting131214** [pantheon5100/pid_diffusion#5]
Hi, author.
Sorry to disturb youin your busy schedule. I'm reading this paper which gives me great inspiration. I admire your experiment very much. The paper mentions that images can be generated through a single-step inference process. However, when I follow the prompts in this code repository, I found the results are full of noise but not clear images.  I was wondering if you could kindly offer some guidance or clarification on my questions.
![samples_first_batch](https://github.com/user-attachments/assets/f3ce9fd4-108d-45b9-9222-399ba8a6b9aa)

**Comment by pantheon5100** [comment]
Hi, if you're not in a hurry, I'll respond to you in a few weeks.

**Comment by fyting131214** [comment]
Thank you very much for your response. Sorry, it's my fault, and I have solved the problem.  I initially ran with the default model path ./model_zoo/edm-cifar10-32x32-uncond-vp.ckpt(line 97 of 'image_sampling.sh'), and then modified to ./model_zoo/pid_cifar.pt. I found that both generated noisy images yesterday. However, I tried again with pid_cifar.pt just now, and the results are clear images. It seems there was an issue with my file transfer and the results were not be overwritten, which caused the error. Maybe it would be great to change the default prompt of line 97 of 'image_sampling.sh'.
